docs/rules/padding-line-between-statements.md

```diff
@@ -42,6 +42,7 @@ You can supply any number of configurations. If a statement pair matches multipl
     - `"any"` just ignores the statement pair.
     - `"never"` disallows blank lines.
     - `"always"` requires one or more blank lines. Note it does not count lines that comments exist as blank lines.
+    - `{number}` (minimum: 1) requires a specified number of blank lines.
```

Comment:
so if I'm not misunderstanding, the above options:

* "any" is the same as `0` (as the number of  blank lines is always >= 0)
* "always" is the same as `1` 

it can be used to simplify the original logic somehow. 😄

Reply:
@aladdin-add What about backward compatibility support for those who are currently using the old view?

Reply:
it should not affect backward compatibility, just a suggestion to simply the implementation.